app/views/admin/account_actions/new.html.haml

```diff
@@ -1,6 +1,13 @@
 - content_for :page_title do
   = t('admin.account_actions.title', acct: @account.pretty_acct)
 
+- if @account.suspended?
+  .flash-message.alert
+    = t('admin.account_actions.already_suspended')
+- elsif @account.silenced?
+  .flash-message.warn
+    = t('admin.account_actions.already_silenced')
```

Comment:
Having this information is good, but I would probably not use the flash styles, especially those indicating an error.

Reply:
Yeah, wasn't a fan of reusing flash styling but wasn't sure how else would look good to style it in an attention grabbing way